admin/test/config/database.yml

```diff
@@ -0,0 +1,18 @@
+localhost:
+  username: 'sa'
+  password: 'Mtc-D3v.5ql_S3rv3r'
+  host: 'localhost'
+  port: '1433'
+  database: 'mtc'
+dev:
+  username: 'sa'
+  password: 'Mtc-D3v.5ql_S3rv3r'
+  host: ''
+  port: '1433'
+  database: 'mtc'
+test:
+  username: 'sa'
+  password: 'Mtc-D3v.5ql_S3rv3r'
+  host: ''
+  port: '1433'
+  database: 'mtc'
```

Comment:
@sinhaamit1980 Is there a need for this file now? As your setting vars in the env.rb now?